source/opt/set_spec_constant_default_value_pass.cpp

```diff
@@ -200,15 +240,36 @@ Pass::Status SetSpecConstantDefaultValuePass::Process(ir::Module* module) {
     }
     if (!spec_inst) continue;
 
-    // Search for the new default value for this spec id.
-    auto iter = spec_id_to_value_.find(spec_id);
-    if (iter == spec_id_to_value_.end()) continue;
+    // Get the default value bit pattern for this spec id.
+    std::vector<uint32_t> bit_pattern = [this, &spec_inst, &spec_id,
```

Comment:
Changed back to normal way. Actually all the rest of the related code is in an if-else branch, so the scope is well limited. Originally I want to avoid copy, I just realized rvalue assignment will do the job for me so I can just assign to the bit_pattern vector.

bit_pattern actually cannot be constant, we do std::move() it later.